src/components/CustomerQuota/ShowFullListToggle.tsx

```diff
@@ -42,12 +42,8 @@ export const ShowFullListToggle: FunctionComponent<{
       >
         <View style={styles.showFullListToggleWrapper}>
           <View style={styles.showFullListToggleBorder} />
-          <Ionicons
-            name={
-              isShowFullList
-                ? "ios-arrow-dropup-circle"
-                : "ios-arrow-dropdown-circle"
-            }
+          <AntDesign
+            name={isShowFullList ? "upcircle" : "downcircle"}
```

Comment:
Are we changing our icon pack?

Reply:
yea, the specific icon used for this disappear from expo @expo/vector-icons. The chosen icon from AntDesign was from discussion with Eida in sally!